src/main/java/com/monov/student/service/StudentService.java

```diff
@@ -1,50 +1,53 @@
 package com.monov.student.service;
 
+import com.monov.commons.dto.ItemIds;
+import com.monov.commons.dto.StudentDTO;
 import com.monov.student.entity.Student;
 import com.monov.student.repository.StudentRepository;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Service
 @Slf4j
 public class StudentService {
 
-    private static final String API_GATEWAY = "API-GATEWAY";
-    private static final String COURSES_ENDPOINT = "courses";
-
     @Autowired
     private StudentRepository studentRepository;
 
-    public Student saveStudent(Student student) {
+    public StudentDTO saveStudent(Student student) {
         log.info("Inside saveStudent method in StudentService");
-        return studentRepository.save(student);
+        return convertToStudentDTO(studentRepository.save(student));
     }
 
-    public Student findStudentById(Long studentId) {
+    public StudentDTO findStudentById(Long studentId) {
         log.info("Inside findStudentById method in StudentService");
-        return studentRepository.findByStudentId(studentId);
+        return convertToStudentDTO(studentRepository.findById(studentId).get());
+    }
+
+    public List<StudentDTO> findAllStudents(){
+        return convertToStudentDTOs(studentRepository.findAll());
     }
 
-    public List<Student> findAllStudents(){
-        return studentRepository.findAll();
+    public List<StudentDTO> findStudentsByIds(ItemIds studentIds) {
+        return convertToStudentDTOs(studentRepository.findAllById(studentIds.getIds()));
```

Comment:
Discussed this with author. Will be fixed.

Reply:
Will ne handled in next PR (About exception and error handling ) which will point to this one in the description/comments